src/js/components/PodInstancesTable.js

```diff
@@ -357,9 +357,10 @@ class PodInstancesTable extends React.Component {
         data={this.getTableDataFor(instances, filterText)}
         expandAll={!!filterText}
         getColGroup={this.getColGroup}
+        itemHeight={TableUtil.getRowHeight()}
         onCheckboxChange={this.handleItemCheck}
-        sortBy={{prop: 'startedAt', order: 'desc'}}
```

Comment:
I don't know if there is a "common" way of sorting tables. I can change this back.
I was just surprised to see the items suddenly appear in reverse order in the tests (for the same `startedAt` value) and I thought of sorting them by something more unique, like the `name`, just in case two `startedAt` timestamps are identical.